stroyprombeton/tests/tests_selenium_admin.py

```diff
@@ -224,6 +224,7 @@ def test_tree_fetch_data(self):
                          .find_elements_by_class_name('jstree-leaf'))
         self.assertGreater(len(node_children), 10)
 
+    @unittest.skip
```

Comment:
Please create a todo for this or remove it

Reply:
already done it at https://github.com/fidals/stroyprombeton/pull/572/files#diff-537c184454430fc9c494fb76f946dfd2R228